crafter-search-batch-indexer/src/main/java/org/craftercms/search/batch/impl/AbstractBinaryFileBatchIndexer.java

```diff
@@ -70,10 +70,12 @@ protected void doSingleFileUpdate(String indexId, String siteName, ContentStoreS
             doDelete(indexId, siteName, path, updateStatus);
         } else {
             Content binaryContent = contentStoreService.getContent(context, path);
-            if (binaryContent.getLength() > maxFileSize) {
-                logger.info("Skipping large binary file @ {}", path);
-            } else {
-                doUpdateContent(indexId, siteName, path, binaryContent, updateDetail, updateStatus, metadata);
+            if (binaryContent != null && binaryContent.getLength() > 0) {
+                if (binaryContent.getLength() > maxFileSize) {
+                    logger.info("Skipping large binary file @ {}", path);
```

Comment:
@sumerjabri @jdrossl shouldn't this be a warning?

Reply:
Yeah, I think you're right.

@jdrossl please upgrade to WARNING